src/store/modules/wallet.js

```diff
@@ -152,10 +153,20 @@ const getters = {
   getLatestEventTimestamp: state =>
     state.events[0]?.timestamp &&
     new Date(state.events[0]?.timestamp).getTime(),
+  getEventLastSeenTs: state => state.eventLastSeenTs,
   getHasUnseenEvents: state =>
     state.eventLastSeenTs &&
     state.events[0]?.timestamp &&
     state.eventLastSeenTs < new Date(state.events[0]?.timestamp).getTime(),
+  getNotificationCount: (state, getters) => {
+    const count = state.events.findIndex(
+      e => state.eventLastSeenTs > new Date(e.timestamp).getTime()
+    );
```

Comment:
We need to filter away our own purchase event from notification
a good filter would be sender = NFT API and receiver = user self wallet
Maybe should refactor and add `eventType` into store instead of in UI component